pallets/nfts/src/tests.rs

```diff
@@ -1048,7 +1055,7 @@ fn set_collection_owner_attributes_should_work() {
 				(Some(0), AttributeNamespace::CollectionOwner, bvec![1], bvec![0]),
 			]
 		);
-		assert_eq!(Balances::reserved_balance(account(1)), 10);
+		assert_eq!(Balances::reserved_balance(account(1)), 10 + 1); // 10 - attribute deposit, 1 - balance deposit
```

Comment:
We don't need the comments here, should be solved in a PR to sdk where we fix the test variables.